src/zenml/enums.py

```diff
@@ -86,3 +86,9 @@ def plural(self) -> str:
             return "container_registries"
 
         return f"{self.value}s"
+
+
+class ContextTypes(Enum):
```

Comment:
I'm not sure whether this will be confusing given the name conflict with our `StepContext`, maybe it makes sense to somehow make it explicit that we're talking about TFX contexts here

Reply:
You are absolutely right. In an ideal case, I would even avoid using the word `Context` here. Do you have any recommendations for a possible alternative? `MLMDPropertyTypes` perhaps? I can go ahead and refactor the naming of the additional parts accordingly as well.

Reply:
Mhm that's a hard one. `MLMDContextTypes` maybe? I quite like the MLMD prefix, but not sure if `Property` is a little misleading (it is a context after all in the MLMD language). But feel free to go with either, you're the expert and I feel everything is okay as long as it contains a TFX/MLMD reference to differentiate it from our `StepContext`

Reply:
I have ended up changing it to `MetadataContextTypes`. I wanted to use something which is distinguishable from the `StepContext` and something that is general (and not tied to a package name like `ml_metadata`) for future proofing.

Reply:
Love it, good to go from my side now!